src/components/RatingsReviews/RatingBreakdown.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+
+function RatingBreakdown ({ meta }) {
+
+  function avgRating() {
+    var sum = 0;
+    var totalReviews = 0;
+    for (var starRating in meta.ratings) {
+      sum += starRating * parseInt(meta.ratings[starRating])
+      totalReviews += parseInt(meta.ratings[starRating])
+    }
+    return Math.round((sum / totalReviews) * 10) / 10
```

Comment:
Just wondering why you used Math.round * 10 and then divide the whole thing rather than a "toFixed" js function

Reply:
I did not know about this method, but it seems way more appropriate for what I'm trying to do. Updated to `return (sum / totalReviews).toFixed(1)`

Thanks for the tip!